VMEncryption/volume-notif-svc/UdevVolNotif.c

```diff
@@ -0,0 +1,222 @@
+#include <libudev.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+#include <time.h>
+#include <errno.h>
+#include <signal.h>
+#include <getopt.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+
+#define LOG_PRIORITY 3
+#define LOG_LEVEL 3
+#define LOG_TO_CONSOLE 1
+
+#define LOG_DIRETORY "/tmp"
+#define LOG_FILE_TEMPLATE "%s/ade_vol_notif-%s.log"
+
+// TODO: Make this a parameter to the program
+static char log_file_path[1024];
+
+void custom_log(const char *format, ...) {
+    FILE *log_file = fopen(log_file_path, "a");
```

Comment:
Do we want to rotate the log after certain size

Reply:
Actually, what I am thinking is to do syslog instead of files. File could be used for debugging purposes.